react-game/src/components/TrialRow/EnteredNumber.js

```diff
@@ -0,0 +1,23 @@
+import { forwardRef } from 'react';
+import classes from './Number.module.css';
+
+const EnteredNumber = forwardRef(({ enableState, clearScreen }, ref) => { //ref is used to forward the input's number to the TrialRow component 
+
+    //to prevent user from entering texts, only numbers are allowed
+    document.addEventListener('keypress', (e) => {
+        var key = e.keyCode;
+        if (key < 48 || key > 57) {
+            e.preventDefault();
+        }
+    });
+
+    //to remove the last number from the input field if the screen needs to be cleared
```

Comment:
No, lets try to clear the input value instead of use 2 inputs

Reply:
done